sources/packages/api/src/route-controllers/institution-locations/institution-locations.controller.ts

```diff
@@ -88,7 +89,7 @@ export class InstitutionLocationsController extends BaseController {
   @Get()
   async getAllInstitutionLocations(
     @UserToken() userToken: IUserToken,
-  ): Promise<any> {
+  ): Promise<InstitutionLocationsDetails[]> {
```

Comment:
API should return only specific DTO objects to avoid that changes on other layers consuming the same object expose more fields than what the API should be exposing.